### PR TITLE
General docs fixup

### PR DESCRIPTION
**Pull request by fukiame**
so ye me fix random small stuff :ded: :bwaa:

### CODE REVIEW

**Comment by fukiame** [comment]
i forgor to draft this :skul: not done yet

**Comment by fukiame** [comment]
ig thats it for this pr, it should be ready for review now iirc

File: wiki.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
-<strong>Building Android</strong>
+## Building Android
 
 Building Android from start to finish (WIP)
 
 ## Getting started
 
-Earlier, Cirrus CI gave the ROM Builders community servers for OSS developement of android builds, as these builds need a lot of resources that most people don't have lying around, like double digit RAM requirements(GB), triple digit storage requirement(GB), a capable processor, a specific linux environment, etc. 
+Earlier, Cirrus CI gave the ROM Builders community server for OSS developement of Android builds, as these builds need a lot of resources that most people don't have lying around, like double digit RAM requirements (GB), triple digit storage requirements (GB), a capable processor, a specific Linux environment, etc.
```
Comment (PugzAreCute) [inline wiki.md]: server should be plural

File: wiki.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
-<strong>Building Android</strong>
+## Building Android
```
Comment (PugzAreCute) [inline wiki.md]: Id say this line is the same as the next. Append from start to finish here and nuke the next line.

Don't say (wip)

File: wiki.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
+Earlier, Cirrus CI gave the ROM Builders community server for OSS developement of Android builds, as these builds need a lot of resources that most people don't have lying around, like double digit RAM requirements (GB), triple digit storage requirements (GB), a capable processor, a specific Linux environment, etc. 
 
-Cirrus CI has discontinued ROM Builders around Nov 2023. Now, [crave.io](https://crave.io) is providing Build Servers with a pretty similar Queue System. It is also a lot more flexible as it allows custom commands, has docker image support, allows entering into build storage without build running, etc.
+Cirrus CI has discontinued ROM Builders around Nov 2023. Now, [crave.io](https://crave.io) is providing Build Servers with a pretty similar Queue System. It is also a lot more flexible as it allows custom commands, has Docker image support, allows entering into build storage without running a build, etc.
```
Comment (PugzAreCute) [inline wiki.md]: Nit: starting a build

File: wiki/Building_101.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ You must first install it on your server.
 
 There are also some additional dependencies, which are usually covered by the `base-devel` package group of your linux distribution.
 
-ArchLinux:
+Arch Linux:
 ```
 sudo pacman -S repo base-devel git-lfs
```
Comment (PugzAreCute) [inline wiki/Building_101.md:18]: Ik I wrote this line

But a LOT more repos are needed iirc

Along with a symlink to ncursesw.so from libncurses.so.5

File: wiki/Debugging.md [hunk 5]
```diff
@@ -61,7 +60,8 @@
     ```
 
 ### Using FBCon
-FBCon enables kernel output to the display. Use this as a last resort if the above mentioned methods dont work
+
+FBCon enables kernel output to the display. Use this as a last resort if the above mentioned methods don't work.
```
Comment (PugzAreCute) [inline wiki/Debugging.md:64]: Note: if you don't select these options from menu configuration it won't work

I'm not near a pc to tell the menu configuration entries

But if you can change this to what all options in menuconfig I'll be grateful

**Review by PugzAreCute — CHANGES_REQUESTED** [review]
A few changes please

File: wiki/Debugging.md (hunk 5 above)
Comment (fukiame) [reply wiki/Debugging.md:64]: about this one... uhh, i dont have any pc rn and i dont have kernel source cloned on my phone, so cant :sorry:

**Comment by fukiame** [comment]
sorry for the late reply was busy doing some works
regarding the Arch Linux packages, it might be a good idea to refer to the [aosp-devel](https://aur.archlinux.org/packages/aosp-devel) or [lineageos-devel](https://aur.archlinux.org/packages/lineageos-devel) AUR meta packages since they seems to be adequete enough in most case
regarding the menuconfig options, unfortunately i dont have access to any PC to check the menuconfig on, so i cant amend it atm

File: wiki/Debugging.md (hunk 5 above)
Comment (PugzAreCute) [reply wiki/Debugging.md:64]: It's alright

**Review by PugzAreCute — APPROVED** [review]
LGTM. I'll fix the rest later

**Review by sounddrill31 — APPROVED** [review]

**Comment by sounddrill31** [comment]
lgtm, great job

**Pull request merged**